src/model/AirLoopHVACOutdoorAirSystem_Impl.hpp

```diff
@@ -121,12 +124,9 @@ namespace model {
       REGISTER_LOGGER("openstudio.model.AirLoopHVACOutdoorAirSystem");
 
       boost::optional<ModelObject> controllerOutdoorAirAsModelObject() const;
-      std::vector<ModelObject> oaComponentsAsModelObjects() const;
-      std::vector<ModelObject> reliefComponentsAsModelObjects() const;
```

Comment:
The actual method oaComponents() already returns a vector of ModelObject so this is pointless